source/speech/__init__.py

```diff
@@ -144,13 +144,22 @@
 import synthDriverHandler
 import config
 from .speech import initialize as speechInitialize
+from .sayAll import initialize as sayAllInitialize
```

Comment:
This means the order has changed subtly, before sayAll was imported after speechInitialize() was called. Does it matter?

Reply:
Matter in what way? The issue with uninstalling addons is still fixed.

Reply:
I'm just pointing out that there is a subtle change in import / initialization order. Because the initialization code used to be at module level, it was run on import, now the import and initialization code is interleaved. There may be hard to predict consequences. It might be worth looking at that and trying to determine if there are any ways it could cause import problems. If we think there is a risk, then we can maintain the previous import / initialization order by moving the imports to within the package `initialize` method (prior to using `sayAllInitialize`)

Reply:
sayAll is no longer dependent on speech so I would think not. The imports all happen in the same order, it's just the initialization happens after sayAllInitialize instead of between `speechInitialize` and `sayAllInitialize`. My risk assessment is as follows:

Importing `sayAllInitialize` earlier, before initializing `speech`, could be a risk because:

1. importing `sayAll` might (eventually) depend on something in speech - this should not be a risk because the import order hasn't changed
1. importing `sayAll` might directly depend on something in speech being initialized - this is not a risk because all direct speech dependencies are dependency injected on initialisation. 
1. module level imports in `sayAll` could be affected if they are dependent on speech being initialized - this is not a risk as all the imports in `sayAll` are also in `speech` or `speech\__init__.py` (other than queueHandler, winKernel, which happen earlier elsewhere)

Initializing `speech` later, after importing `sayAll`, shouldn't be a risk because speech was never dependent on importing `sayAll`.